contracts/facets/ViewStateFacet.sol

```diff
@@ -44,6 +44,33 @@ contract ViewStateFacet {
         return profiles;
     }
 
+    /// @notice Gets statistics for each archaeologist
+    /// Contains a list of sarcoIds for each category. We could simply return the counts of the
+    /// arrays but we are already storing the lists of sarcoIds so we may as well use them.
+    /// @param addresses The list of archaeologist addresses
+    /// @return The list of archaeologist statistics
+    function getArchaeologistsStatistics(address[] memory addresses)
+        external
+        view
+        returns (LibTypes.ArchaeologistStatistics[] memory)
+    {
+        LibTypes.ArchaeologistStatistics[]
+            memory statsList = new LibTypes.ArchaeologistStatistics[](
+                addresses.length
+            );
+
+        for (uint256 i = 0; i < addresses.length; i++) {
+            statsList[i] = LibTypes.ArchaeologistStatistics(
+                addresses[i],
+                s.archaeologistSuccesses[addresses[i]],
+                s.archaeologistCancels[addresses[i]],
+                s.archaeologistAccusals[addresses[i]]
+            );
+        }
```

Comment:
note -- see my comments on discord -- I think we just need the lengths of the mappings for these, and not the sarcoIds. Also think the original successes mapping can be removed (and any view state functions associated with it).